packages/@expo/cli/src/utils/env.ts

```diff
@@ -66,10 +66,6 @@ class Env {
   get EXPO_NO_CACHE() {
     return boolish('EXPO_NO_CACHE', false);
   }
-  /** Enable the experimental interstitial app select page. */
-  get EXPO_ENABLE_INTERSTITIAL_PAGE() {
```

Comment:
Replace the entire env with `EXPO_NO_REDIRECT_PAGE`, the feature should still be capable of disabling, but off by default.

The naming:

- We use the format `EXPO_NO_*` for enabled by default and `EXPO_*` (ex `EXPO_DEBUG`) for disabled by default.
- Use 'redirect' instead of 'interstitial'. 
  - There is no autocorrect for or validation for environment variables, so misspelled words can be hard to debug. We use env vars that are easy to spell correctly.
  - Android uses the phrase 'disambiguation dialog', so we aren't unifying.

Reply:
Also need to update the docs https://github.com/expo/expo/blob/main/docs/pages/workflow/expo-cli.md (bottom)